unstract/sdk1/tests/test_sampling_strip.py

```
@@ -0,0 +1,271 @@
+        "model_id": "arn:aws:bedrock:us-east-1:0:application-inference-profile/abcd",
+        "temperature": 0.5,
+    }
+    out = _strip_deprecated_sampling_params(inp)
+    assert "temperature" not in out
+
+
+def test_strip_skipped_when_both_fields_opaque_and_logs_debug(
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    """Documented limitation: with no model id in any field, the strip is
+    a no-op and emits a debug breadcrumb so the upstream 400 is traceable."""
```

Ruff (0.15.12)

[warning] 1 blank line required between summary line and description

(D205)